packages/react-components/src/templates/UserProfileHeader.tsx

```diff
@@ -196,127 +202,144 @@ const UserProfileHeader: React.FC<UserProfileHeaderProps> = ({
   const { isOwnProfile } = useContext(UserProfileContext);
 
   return (
-    <header css={[containerStyles]}>
-      <section css={personalInfoStyles}>
-        <div>
-          <div css={nameHeaderStyles}>
-            <div css={{ display: 'flex' }}>
-              <Display styleAsHeading={2}>{displayName}</Display>
-              {degree ? (
-                <Display styleAsHeading={2}>, {degree}</Display>
-              ) : isOwnProfile ? (
-                <div css={{ color: tin.rgb }}>
-                  <Display styleAsHeading={2}>, Degree</Display>
-                </div>
-              ) : null}
+    <>
+      {alumniSinceDate && (
+        <Toast accent="info">
```

Comment:
It seems there's a problem when the text is too large inside the `Toast` for mobile view because it has a fixed height (https://github.com/yldio/asap-hub/blob/master/packages/react-components/src/organisms/Toast.tsx#L55)

<img width="250" src="https://user-images.githubusercontent.com/16595804/196497429-06e3c29e-07cd-4471-9288-e76340a52544.png" />